### PR TITLE
feat: add auth session timeouts

### PR DESCRIPTION
**Pull request by hf**
Adds the Auth session timeouts configs to the CLI.

### CODE REVIEW

**Review by kamilogorek — APPROVED** [review]
ref: https://github.com/supabase/gotrue/pull/1288
ref: https://github.com/supabase/infrastructure/pull/15533/

**Comment by hf** [comment]
Don't merge until fully released!

**Comment by kamilogorek** [comment]
image
:D

File: internal/utils/config.go [hunk 1]
```diff
@@ -343,6 +345,11 @@ type (
 		ApiKey           string          `toml:"-" mapstructure:"api_key"`
 	}
 
+	sessions struct {
+		Timebox           float64 `toml:"timebox"`
+		InactivityTimeout float64 `toml:"inactivity_timeout"`
```
Comment (sweatybridge) [inline internal/utils/config.go:350]: Please also update [init_config.toml](https://github.com/supabase/cli/blob/main/internal/utils/templates/init_config.toml) and init_config_test.toml with example values.

**Comment by sweatybridge** [comment]
Is this safe to merge?

**Comment by sweatybridge** [comment]
closing due to staleness

**Comment by hf** [comment]
Sorry about that. Addressed the comments!

**Pull request closed without merging**